src/site/_includes/macros/tagcard.njk

```diff
@@ -0,0 +1,34 @@
+{% macro tagcard(collectionItem) %}
+
+{% set url = '/tags/'+collectionItem.key %}
+{% set data = collectionItem.data %}
+{% set thumbnail = data.thumbnail or data.hero or null %}
+{% set alt = data.alt or '' %}
+
+<div class="w-card" role="listitem">
+  <article class="w-card-base">
+    {% if thumbnail %}
+      <div class="w-card-base__cover w-card-base__cover--with-image }}">
+        <a href="{{url}}" class="w-card-base__link" tabindex="-1" aria-hidden="true">
```

Comment:
Similar to the authorcard, can you add a comment explaining that we hide the image from screen readers here because we often don't provide descriptive alt text.